docker-compose.yml

```diff
@@ -33,23 +33,17 @@ services:
         volumes:
             - redis-data:/data
 
-    warp-socks:
-        image: monius/docker-warp-socks:latest
-        privileged: true
+    warp:  # new warp service
+        image: aleskxyz/warp-svc:latest
```

Comment:
This image provides amd64 arch only. However, the docker compose file is expected to be used at least on  arm64 and amd64.

Reply:
thank you for your comment.